authentication/management/commands/validate_user_emails.py

```diff
@@ -0,0 +1,123 @@
+"""Management command for starting the verification of user emails"""
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+from django.db.models import Exists, OuterRef
+
+from authentication import email_validation_api
+from authentication.models import EmailValidation
+import requests
+
+
+User = get_user_model()
+
+
+def get_unprocessed_validation_lists():
+    """Return a QuerySet for unprocess"""
+    return (
+        EmailValidation.objects.filter(result__isnull=True)
+        .values_list("list_name", flat=True)
+        .distinct()
+    )
+
+
+class Command(BaseCommand):
+    """Verify user emails"""
+
+    help = "Verify user emails"
+
+    def add_arguments(self, parser):
+        subparsers = parser.add_subparsers(dest="subcommand", required=True)
+        start_parser = subparsers.add_parser("start")
+        start_parser.add_argument(
+            "--db-chunk-size",
```

Comment:
I tried running `./manage.py validate_user_emails start --db-chunk-size 1` and got '>' not supported between instances of 'str' and 'int'

Reply:
I removed the option